en/peripherals/camera.md

```diff
@@ -33,17 +33,24 @@ Mode | Description
 
 ## Trigger Hardware Configuration {#hardware_setup}
 
-You can choose which pins to use for triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
+The pins used to trigger image capture for GPIO, PWM or Seagull-based triggering (i.e. when not using a MAVLink camera) are set using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
 The default is 56, which means that trigger is enabled on *FMU* pins 5 and 6.
 
 > **Note** On a Pixhawk flight controller that has both FMU and I/O boards these FMU pins map to `AUX5` and `AUX6` (e.g. Pixhawk 4, CUAV v5+). 
   On a controller that only has an FMU, the pins map to `MAIN5` and `MAIN6` (e.g. Pixhawk 4 mini, CUAV v5 nano).
   At time of writing triggering only works on FMU pins - you can't trigger a camera using pins on the I/O board.
 
 <span></span>
-> **Warning** With `TRIG_PINS` set to its **default** value of 56, you can use the AUX pins 1, 2, 3 and 4 as actuator outputs (for servos/ESCs).
-  Due to the way the hardware timers are handled (1234 and 56 are 2 different groups handled by 2 timers), this is the ONLY combination which allows the simultaneous usage of camera trigger and FMU actuator outputs.
-  **DO NOT CHANGE THE DEFAULT VALUE OF `TRIG_PINS` IF YOU NEED ACTUATOR OUTPUTS.**
+> **Warning** With `TRIG_PINS=56` (default) you can use the AUX pins 1 to 4 as actuator outputs (for servos/ESCs).
+  With `TRIG_PINS=78`, you can use the AUX pins 1-6 as actuator outputs.
+  Any other combination of pins can be selected, but this will disable use of the other FMU pins as outputs.
+  
+Some cameras can provide image capture signalling to a flight controller camera-capture pin (this enables more accurate image timestamp than the time that the trigger command is sent).
+- Pixhawk FMUv5x boards use the board-specific camera capture pin (PI0).
+- Other board use FMU PWM pin 6 (hardcoded) for camera capture.
+
+> **Note** The camera driver and camera capture driver emit the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message (with timestamp and image sequence number) when image capture is triggered and signaled, respectively.
```

Comment:
@bkueng FYI here I added info abut the camera trigger. Seem correct to you? Looking at the code it looks like the camera driver just updates the uorb topic and that ultimately causes the message to be omitted.

It seems to me that a MAVLink camera should emit CAMERA_TRIGGER to.

Anyway, this make sense?

Reply:
Yes that helps, thanks.
The fmu will emit CAMERA_TRIGGER in any case though: https://github.com/PX4/Firmware/blob/master/src/drivers/camera_trigger/camera_trigger.cpp#L330.
Probably worth clarifying in the mavlink spec as well.

Reply:
Hmmm. On further thought and reading I think the behaviour here will remain undefined - the commands that PX4 uses are not deprcated - do_trigger_control etc - but they are very much "old style". The new camera API uses new ones. 

The good news here is that I think it doesn't matter. A MAVLink camera should emit CAMERA_IMAGE_CAPTURED on every capture, with a timestamp. A geotagging system should simply use that by preference if both messages are present. Make sense?

Reply:
I really lack the background on all these messages to give you a good answer, but you're probably right. Daniele should have the overview, but he's out this week.

Reply:
OK. Gus pretty much said that the behaviour for the old commands on a mavlink camera is undefined. That said, he did indicate that if the camera did obey the command then it would emit `CAMERA_IMAGE_CAPTURED`. I think we need to get those old commands deprecated and support for the new commands added to PX4.